android/composeui/build.gradle

```diff
@@ -68,7 +68,9 @@ dependencies {
 
 mavenPublishing {
     publishToMavenCentral(SonatypeHost.CENTRAL_PORTAL)
```

Comment:
sorry to add more repetition here - at some point we should move this to a convention plugin and just apply it once so everyone gets it automatically.